shell/platform/darwin/ios/ios_gl_render_target.mm

```diff
@@ -116,7 +115,7 @@
   glBindRenderbuffer(GL_RENDERBUFFER, colorbuffer_);
   FML_DCHECK(glGetError() == GL_NO_ERROR);
 
-  if (![context_.get() renderbufferStorage:GL_RENDERBUFFER fromDrawable:layer_.get()]) {
+  if (!onscreen_context_->BindRenderbufferStorage(GL_RENDERBUFFER, std::move(layer_))) {
```

Comment:
I don't think we want to move `layer_` here. We might need it again later.

Reply:
good catch, got a bit too enthusiastic with my use of std::move :)